modules/install/pages/upgrade.adoc

```diff
@@ -4,13 +4,28 @@
 [abstract]
 {description}
 
+[#before-you-upgrade]
+== Before You Upgrade
+
+Before upgrading, consider the following version compatibility concerns.
+
 // So long as upgrading from 6.x is supported, this notice will need to stay in some form in each new release.
-.Upgrading to Version 7.x with older versions of .NET SDK
-[WARNING]
-====
+=== Upgrading to Version 7.x With Earlier Versions of .NET SDK
 
-If you are upgrading to Couchbase *6.5/6.6 -> 7.x+*  whilst using .NET SDK prior to 3.2.9 then make sure your cluster is *not* running in `mixed mode` (i.e. some nodes are using IPv4 addressing while some are using IPv6) before issuing any write operations on the cluster.
-====
+When upgrading from Couchbase 6.5 or 6.6 to 7.0 or later determine if both of the following are true:
+
+* You use a version of the .NET SDK prior to 3.2.9
+* Your cluster is in mixed mode networking where some nodes use IPv4 addressing and others use IPv6. See xref:manage:manage-nodes/manage-address-families.adoc#changing-address-family-to-IPv6[Changing Address Family] for steps to determine if your cluster is running in this mode.
+
+Using a version of the .NET SDK prior to 3.2.9 with mixed mode network addressing can cause issues with write operations. 
+Before upgrading, resolve the mixed mode networking issue.
+
+=== Upgrading From Pre-7.1 Versions of Couchbase Server
+
+You cannot upgrade directly from a version of Couchbase Server earlier than 7.1 to version 7.6 or later. 
```

Comment:
I think the correct statement is the following:
```
You cannot upgrade directly from a version of Couchbase Server earlier than 7.1 to version 7.2.4 or later.
```

For example 
you can upgrade 6.6 -> 7.2.3
you can't upgrade 6.6 -> 7.2.4